doc: Fix various documention issues (#1566)
aristanetworks/anta: docs/cli/get-tests.md

```

!!! tip

>     By default, `anta get commands` returns the commands from every test built in to ANTA.

Reviewer: I feel like built-in is more correct fro what we want ot exprss

whart about

```suggestion
    By default, `anta get commands` retrieves commands from all ANTA's built-in tests.
```
